packages/amplify-category-function/src/provider-utils/awscloudformation/utils/packageLayer.ts

```diff
@@ -31,13 +56,29 @@ export function packageLayer(context, resource) {
 
     zip.pipe(output);
     glob
-      .sync(resourcePath + '/lib/*')
+      .sync(resourcePath + '/!(dist)')
       .filter(folder => fs.lstatSync(folder).isDirectory())
       .forEach(folder => zip.directory(folder, path.basename(folder)));
     zip.finalize();
   });
 }
 
+export async function hashLayerDir(layerPath: string): Promise<any> {
+  const hashOptions = {
+    folders: { exclude: ['.*', 'dist'] },
+    files: { exclude: ['README.txt', 'layer-parameters.json', 'parameters.json', '*-awscloudformation-template.json'] },
```

Comment:
If the customer just changed the permissions in CFN, wont that be a problem because we will bump a version locally as hash will change.

Reply:
Hmm yeah that's true but won't we run into problems if say, they manually update the compatible runtimes of the layer? Maybe we're okay with not covering that edge case. I guess the alternative is to include a hash of just the lambdalayer part of the CFN rather than the entire CFN

Reply:
I see we're still excluding the CFN template (which I think is okay) but we need to document that you should not make manual changes to the layer CFN, or if you do you may also need to update the version metadata that we store

Reply:
What's the best way to document this? Possible options that I can think of:
In a comment inside the CFN template
A message from the CLI on Layer creation
In the docs

Reply:
I think it's better if we can put these in the docs not to make manual changes to CFN file for now.

Reply:
I would also put a message in the CLI on layer creation and maybe even a reminder in the update message. We can't put comments in the CFN template because strict JSON can't have comments